src/utils/DialogIds.ts

```diff
@@ -42,6 +42,7 @@ export const DialogIds = {
     UpdateTextMsgSetupDialogId: "UpdateTextMsgSetupDialog",
     NotifyDialogId: "NotifyDialog",
     PopupSignInDialogId: "PopupSignInDialog",
+    AdaptiveCardDialog: "AdaptiveCardDialog",
```

Comment:
Change to AdaptiveCardDialogId: "AdaptiveCardDialog" to keep up the naming scheme.

Reply:
Done.